scripts/variantstore/wdl/extract/populate_alt_allele_table.py

```diff
@@ -0,0 +1,86 @@
+import time
+import os
+
+from google.cloud import bigquery
+from google.cloud.bigquery.job import QueryJobConfig
+from google.oauth2 import service_account
+from pathlib import Path
+
+import argparse
+
+client = None
+
+def execute_with_retry(label, sql):
+    retry_delay = [30, 60, 90] # 3 retries with incremental backoff
+    start = time.time()
+    while len(retry_delay) > 0:
+        try:
+            query_label = label.replace(" ","-").strip().lower()
+
+            existing_labels = client._default_query_job_config.labels
+            job_labels = existing_labels
+            job_labels["gvs_query_name"] = query_label
+            job_config = bigquery.QueryJobConfig(labels=job_labels)
+            query = client.query(sql, job_config=job_config)
+
+            print(f"STARTING - {label}")
+            results = query.result()
+            print(f"COMPLETED ({time.time() - start} s, {3-len(retry_delay)} retries) - {label}")
+            return results
+        except Exception as err:
+            # if there are no retries left... raise
+            if (len(retry_delay) == 0):
+                raise err
+            else:
+                t = retry_delay.pop(0)
+                print(f"Error {err} running query {label}, sleeping for {t}")
+                time.sleep(t)
+
+def populate_alt_allele_table(query_project, vet_table_name, fq_dataset, sa_key_path):
+    global client
+    default_config = QueryJobConfig(priority="INTERACTIVE", use_query_cache=True)
+
+    if sa_key_path:
+        credentials = service_account.Credentials.from_service_account_file(
+            sa_key_path, scopes=["https://www.googleapis.com/auth/cloud-platform"],
+        )
+
+        client = bigquery.Client(credentials=credentials,
+                                 project=query_project,
+                                 default_query_job_config=default_config)
+    else:
+        client = bigquery.Client(project=query_project,
+                                 default_query_job_config=default_config)
+
+    os.chdir(os.path.dirname(__file__))
+    alt_allele_temp_function = Path('alt_allele_temp_function.sql').read_text()
+    alt_allele_positions = Path('alt_allele_positions.sql').read_text()
+    first = True if vet_table_name == "vet_001" else False
```

Comment:
Since this function is being called in a scatter in the WDL, there's no guarantee that vet_001 will be the first one to execute.  If this is the flow, it might be better to extract out a separate "create table" task that runs before the scatter and then all the "populates" can be straight inserts